osu.Game.Tests/Editing/Checks/CheckPreviewTimeTest.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Edit.Checks;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Tests.Beatmaps;
+
+namespace osu.Game.Tests.Editing.Checks
+{
+    public class CheckPreviewTimeTest
+    {
+        private CheckPreviewTime check = null!;
+
+        private IBeatmap beatmap = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            check = new CheckPreviewTime();
+            beatmap = new Beatmap<HitObject>
+            {
+                BeatmapInfo = new BeatmapInfo
+                {
+                    Metadata = new BeatmapMetadata { PreviewTime = -1 },
+                    BeatmapSet = new BeatmapSetInfo(new List<BeatmapInfo>
+                    {
+                        new BeatmapInfo
+                        {
+                            DifficultyName = "Test1",
+                            Metadata = new BeatmapMetadata { PreviewTime = 5 },
+                        },
+                        new BeatmapInfo
+                        {
+                            DifficultyName = "Test2",
+                            Metadata = new BeatmapMetadata { PreviewTime = 10 },
+                        },
+                    })
+                }
+            };
```

Comment:
Why is this set if both tests call methods that seem to overwrite this beatmap entirely anyway?